docs/topics/clusterdefinitions.md

```diff
@@ -725,6 +725,7 @@ A cluster can have 0 to 12 agent pool profiles. Agent Pool Profiles are used for
 | customVMTags                                                   | no                                                                   | Specifies a list of custom tags to be added to the agent VMs or Scale Sets. Each tag is a key/value pair (ie: `"myTagKey": "myTagValue"`).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
 | diskEncryptionSetID                                            | no                                                             | Specifies ResourceId of the disk encryption set to use for enabling encryption at rest (ie: `"/subscriptions/{subs-id}/resourceGroups/{rg-name}/providers/Microsoft.Compute/diskEncryptionSets/{diskEncryptionSet-name}"`). More details about [Server side encryption of Azure managed disks](https://docs.microsoft.com/en-us/azure/virtual-machines/windows/disk-encryption).                                                                                                                                                                                                                                                                                                                                                                |
+| ultraSSDEnabled                                            | no                                                             | Enable UltraSSD feature for each node VM. More details about [Ultra disk](https://docs.microsoft.com/en-us/azure/virtual-machines/linux/disks-types#ultra-ssd-preview).                                                                                                                                                                                                                                                                                                                                                                |
```

Comment:
Thanks for adding to masterProfile! Could you also add this documentation note under the `masterProfile` section?